apps/prairielearn/src/tests/courseEditor.test.ts

```diff
@@ -329,6 +349,29 @@ describe('test course editor', { timeout: 20_000 }, function () {
       });
     });
   });
+
+  describe('Copy from another course', function () {
+    beforeAll(createCourseFiles);
+    afterAll(deleteCourseFiles);
+
+    beforeAll(helperServer.before(courseDir));
+    afterAll(helperServer.after);
+
+    beforeAll(async () => {
+      await sqldb.queryAsync(sql.update_course_repository, {
+        course_path: courseLiveDir,
+        course_repository: courseOriginDir,
+      });
+    });
+
+    beforeAll(createSharedCourse);
+
+    describe('verify edits', async function () {
+      publicCopyTestData.forEach((element) => {
+        testEdit(element);
+      });
+    });
+  });
```

Comment:
I put this in a separate `describe` than the other tests, because loading a second course during the other tests broke a bunch of hard-coded question, assessment, and course instance IDs.